src/index.js

```diff
@@ -2,11 +2,10 @@
  * @namespace PIXI
  */
 
-import * as PIXI from 'pixi.js';
 import AfterEffects from './AfterEffects';
 import AEDataLoader from './loader';
 import AEDataInterceptor from './interceptor';
 
-PIXI.AfterEffects = AfterEffects;
-PIXI.AEDataLoader = AEDataLoader;
-PIXI.AEDataInterceptor = AEDataInterceptor;
+window.PIXI.AfterEffects = AfterEffects;
```

Comment:
_"... Lastly, some 3rd-party plugins maybe expecting window.PIXI, so you might have to explicitly expose the global like this, however this is not recommended."_
[Building package](https://github.com/pixijs/pixi.js/wiki/v5-Migration-Guide#bundling-changes)